Sources/Starscream/Compression.swift

```diff
@@ -56,8 +56,9 @@ class Decompressor {
     }
 
     func decompress(_ data: Data, finish: Bool) throws -> Data {
-        return try data.withUnsafeBytes { (bytes:UnsafePointer<UInt8>) -> Data in
-            return try decompress(bytes: bytes, count: data.count, finish: finish)
+        return try data.withUnsafeBytes { (bytes:UnsafeRawBufferPointer) -> Data in
```

Comment:
I don't want to nit pick but could we add space after `bytes:`

Reply:
added a space